<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDlg</class>
 <widget class="QWidget" name="TableDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="MainView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>背景色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_backcolorbtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_bordercheck">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>描边</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_bordercolorbtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_borderlable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>133</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>边框色：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>格子</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>横向个数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>纵向个数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_hnumberspin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_Vnumberspin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_const_H">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>动态控制</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_const_V">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>动态控制</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_H">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddr_const_H">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>18</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_V">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddr_const_V">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>48</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>格线</string>
   </property>
   <widget class="QCheckBox" name="m_hcheck">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>水平</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_vcheck">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>垂直</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线条类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线条颜色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_linecolorbtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="m_linetypecmb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/0.ico</normaloff>:/lineType/images/LineType/0.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L1.ico</normaloff>:/lineType/images/LineType/L1.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L2.ico</normaloff>:/lineType/images/LineType/L2.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L3.ico</normaloff>:/lineType/images/LineType/L3.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L4.ico</normaloff>:/lineType/images/LineType/L4.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
     <property name="icon">
      <iconset resource="../../resourse.qrc">
       <normaloff>:/lineType/images/LineType/L5.ico</normaloff>:/lineType/images/LineType/L5.ico</iconset>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QSpinBox" name="m_TransparentspBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="m_Transparentlabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>171</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>透明度：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
